packages/api/src/controllers/v1/collector/mod.rs

```diff
@@ -0,0 +1,249 @@
+#![allow(unused)]
+use bdk::prelude::*;
+use by_axum::{
+    aide,
+    auth::Authorization,
+    axum::{
+        Extension, Json,
+        extract::{Path, Query, State},
+        routing::{get, post},
+    },
+};
+
+use common::{
+    Result,
+    error::ServiceError,
+    tables::collectors::{
+        Collector, CollectorAction, CollectorByIdAction, CollectorCreateRequest,
+        CollectorGetResponse, CollectorParam, CollectorQuery, CollectorRepository,
+        CollectorSummary, CollectorUpdateRequest
+    },
+};
+
+use sqlx::postgres::PgRow;
+#[cfg(test)]
+mod tests;
+
+#[derive(
+    Debug, Clone, serde::Deserialize, serde::Serialize, schemars::JsonSchema, aide::OperationIo,
+)]
+pub struct CollectorPathParam{
+    agit_id: i64,
+}
+
+#[derive(
+    Debug, Clone, serde::Deserialize, serde::Serialize, schemars::JsonSchema, aide::OperationIo,
+)]
+pub struct CollectorIdPath{
+    id: i64,
+}
+
+#[derive(
+    Debug, Clone, serde::Deserialize, serde::Serialize, schemars::JsonSchema, aide::OperationIo,
+)]
+
+pub struct CollectorByIdPathParam{
+    id: i64,
+    agit_id: i64,
+}
+
+#[derive(Clone, Debug)]
+pub struct CollectorControllerV1 {
```

Comment:
```suggestion
pub struct CollectorController {
```

controller version has already described in directory path.